src/os/bluestore/BlueStore.cc

```diff
@@ -5778,7 +5719,6 @@ int BlueStore::_do_read(
   }
 
   utime_t start = ceph_clock_now();
-  o->flush();
```

Comment:
o->flush cannot be removed. That's because worker can read the buffer of uncommitted transaction...
read request should be serviced after all flush_txns of the onode execute db->submit_transaction_sync